ui/page/send/page.go

```diff
@@ -96,23 +99,30 @@ type selectedUTXOsInfo struct {
 	totalUTXOsAmount int64
 }
 
-func NewSendPage(l *load.Load) *Page {
+func NewSendPage(l *load.Load, isModalLayout bool) *Page {
 	pg := &Page{
 		Load:             l,
 		GenericPageModal: app.NewGenericPageModal(SendPageID),
-		sendDestination:  newSendDestination(l),
-		amount:           newSendAmount(l),
-
-		exchangeRate: -1,
+		Modal:            l.Theme.ModalFloatTitle(values.String(values.StrSend)),
```

Comment:
Don't assign values to both GenericPageModal and Modal.
Instead, create the `pg` variable without assigning both of them and then do this:
```
if isModalLayout {
    pg.Modal = l.Theme.ModalFloatTitle(values.String(values.StrSend))
    pg.GenericPageModal = pg.Modal.GenericPageModal
} else {
    pg.GenericPageModal = app.NewGenericPageModal(SendPageID)
}
```